packages/dds/map/src/interfaces.ts

```diff
@@ -39,7 +39,7 @@ export interface IValueChanged {
  *
  * @remarks When used as a Map, operates on its keys.
  * @legacy
- * @alpha
+ * @public
```

Comment:
These seem like the kinds of interfaces that should be sealed as we might want to add things to them, and don't expect others to implement them. Might be good to do this before they go public, or at the same time:
```suggestion
 * @sealed @public
```

(aslo applies to IDirectoryEvents and IDirectoryValueChanged)

Reply:
Sealed `ISharedDirectory` and `ISharedDirectoryEvents`, too.